src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -40,7 +40,8 @@
             + PREFIX_MAJOR + "Computer Science "
             + PREFIX_RESUME + "jdoe.pdf "
             + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_TAG + "owesMoney\n"
+            + "The status is initialised to be 'new'.";
```

Comment:
Is this line really necessary?🤔

Reply:
And i think we should change the tag to sth else haha. Quite weird for HRs to add a tag called "owesMoney" 😅